### PR TITLE
The big reformat

### PR DESCRIPTION
**Pull request by tymees**
This PR reformats the entire codebase using Black and DJLint for python and templates respectively.

I checked every file after reformat; the black reformat went well, no issues.

The DJLint reformat had a couple problems:
- JS formatting was borked (we already knew this), so I just disabled it. We probably want a backlog issue to enable it
- Some pages were using invalid HTML that I had to correct; a couple of examples:
  - `<ul><p><li>[..]</li></p></ul>`. Those `p`'s aren't allowed there
  - `<\br>` - yeah dunno how this happened
  - Missing closing tags
- `blocktrans` without `trimmed` messed things up BAD. (It effectively disabled formatting, even for a couple of lines after the end tag). So, I added `trimmed` to every blocktrans. 
- Translations were borked, but that was to be expected. I think I fixed all issues. 
- Update: there was also one 'Django-in-CSS' that DJLint broke... Luckily this one I could just disable in-place instead of globally

Lastly, this PR enabled CI based checks to see if someone forgot to format their code :)

### CODE REVIEW

**Comment by tymees** [comment]
Hmmm, the djlint check seems to be checking.... reads log... DSC 2? Wut...

I'll look at that next week, I'm done

**Review by EdoStorm96 — APPROVED** [review]
I'd say it all looks good. Impressive work! We might run into tiny things later ... but on first glance it all looks good!

**Comment by miggol** [comment]
> <\br> - yeah dunno how this happened

Haha, lol

File: main/templates/base/form_buttons.html [hunk 1]
```diff
@@ -3,37 +3,43 @@
...
-                   value="{{ next_text|default:_('Opslaan en volgende stap >>') }}"/>
+            <input class="button button-colored continue-button"
+                   type="submit"
+                   name="save_continue"
+                   value="{{ next_text|default:_("
+                   Opslaan
+                   en
+                   volgende
+                   stap>
+            >") }}" />
```
Comment (miggol) [inline main/templates/base/form_buttons.html:42]: Somethings's wrong here

**Review by miggol — CHANGES_REQUESTED** [review]
Only wonky translation I found was the following:

```pot
#: reviews/templates/reviews/review_detail_sidebar.html:46
#, python-format
msgid "Reviewronde gestart op <br /> %(date_start)s."
msgstr "eviewing round started on %(date_start)s."
```

Went looking for other msgstrs that don't start with a capital letter, and it turns out there's only like 10 of those total. They're all good.

I took the liberty of committing translations fixes directly into this branch. But my other comment still needs looking at.

File: main/templates/base/form_buttons.html (hunk 1 above)
Comment (tymees) [reply main/templates/base/form_buttons.html]: Interesting... Looks like DJLint has a bug when it comes to the `_('[..]')` syntax. 

Should be fixed now, I moved the translated string to a variable and referenced that in the default filter

**Comment by tymees** [comment]
Fun fact, FF crashes if you try to search through the diff of the locale file lol

> I took the liberty of committing translations fixes directly into this branch. But my other comment still needs looking at.

I'm not seeing any commits?

**Review by miggol — APPROVED** [review]
Whoops, forgot to push. It's there now. No other issues.

**Pull request merged**